content/spin/v2/http-trigger.md

```diff
@@ -78,6 +79,12 @@ route = "/users/..."
 component = "user-manager"
 ```
 
+You can use wildcards to match 'patterns' of routes. Spin supports two kinds of wildcard: single-segment wildcards and trailing wildcards.
```

Comment:
These paragraphs (current lines 82-86) should replace the first two sentences of line 68, rather than appearing after the existing content.  That was why I didn't put the text as a GH suggestion - we have an existing discussion of wildcard routes and the new content should be part of that.